db/seeds.rb

```diff
@@ -9,22 +9,25 @@ def make_books
 
 def make_users
   1.upto(5) do |n|
-    User.create!  email: "user#{n}@gmail.com",
-                  role: 0,
-                  password: "password",
-                  organization: "org#{n}"
+    user = User.create  email: "user#{n}@gmail.com",
+                        role: 0,
+                        password: "password"
+    user.projects << Project.find(1 + Random.rand(Project.count))
+    user.save
   end
   1.upto(2) do |n|
-    User.create!  email: "admin#{n}@gmail.com",
-                  role: 1,
-                  password: "password",
-                  organization: "worldreader"
+    user = User.create  email: "admin#{n}@gmail.com",
+                        role: 1,
+                        password: "password"
+    user.projects << Project.find(1 + Random.rand(Project.count))
+    user.save
   end
   1.upto(2) do |n|
-    User.create!  email: "vip#{n}@gmail.com",
-                  role: 2,
-                  password: "password",
-                  organization: "worldreader"
+    user = User.create  email: "vip#{n}@gmail.com",
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.